vembrane/globals.py

```diff
@@ -95,6 +96,47 @@
     name: mod for name, mod in vars(math).items() if not name.startswith("__")
 }
 
+
+_statistics_whitelist = {
```

Comment:
Maybe `_statistics_uselist` or `_statistics_allowlist`, instead?

Also, is this just a security feature, in case unsafe functions sneak into the `statistics` module? Because this mainly sounds like an extra maintenance burden to me... :sweat_smile:

Reply:
Right, I'll rewrite this